test/runtests.jl

```diff
@@ -79,7 +79,8 @@ end
     at = JuLIP.bulk(:Si, cubic=true)
     deleteat!(at, 1)
     JuLIP.set_calculator!(at, JuLIP.StillingerWeber())
-    model = AdiabaticModels.JuLIPModel(at)
+    incl = ones(1,length(at))
+    model = AdiabaticModels.JuLIPModel(at,incl)
```

Comment:
Instead of changing the test. I would add a constructor where the second argument defaults to ones for all degrees of freedom. In the unlikely event that anyone else is using this will ensure their code doesn't break.